### DISCUSSION TITLE
使用订阅链接无法显示hy2节点

### DISCUSSION THREAD

**Discussion by Richardleeeeeee** [immortalwrt/homeproxy#219] [Q&A]
已成功使用，先感谢大佬开发。
使用的机场订阅链接有hy2节点，但是订阅成功后没有相关节点出现。
用clash可以全部显示。
想问问我会不会哪里没有处理好？
请求各位解惑。

**Comment by Caelvyn** [comment]
我也有这个问题，+1

**Comment by Richardleeeeeee** [comment]
> 我也有这个问题，+1

估计没有别人有这个问题，我也没有得到任何回应，如果真不行就选择另外的软件吧。

**Reply by Rangtian** [reply]
> > 我也有这个问题，+1
> 
> 估计没有别人有这个问题，我也没有得到任何回应，如果真不行就选择另外的软件吧。

用Sub-Store转换为通用订阅，再导入HomeProxy，就能正确解析HY2节点了。

**Comment by 1715173329** [comment]
本人没有 hy2 节点，订阅链接的解析是按照官方 schema 做的。如果你的订阅链接不符合官方标准，自然无法处理。更何况你到现在连个订阅的日志都没发过，请人算命吗？
